src/components/bubble/_bubble.scss

```diff
@@ -14,12 +15,17 @@ $include-html: false !default;
 
   .mint-bubble {
     @include component;
+    display: inline-flex;
```

Comment:
`inline-flex` prevents bubble from occupying the whole available horizontal space